cloudflare-docs-theme/src/components/Link.tsx

```diff
@@ -10,10 +10,10 @@ type LinkProps = {
   [x: string]: any // To improve types, might want to inherit from  GatsbyLinkProps<TState> instead
 }
 
-// replace anything with /workers or / prepended to just /workers or whatever is in publicPath
-const stripWorkers = (url: string, path: string) => {
-  const regex = new RegExp('^(\\/' + path + '){1,3}')
-  return url.replace(regex, '').replace(/^\/(?!\/)/, '/' + path + '/')
+// replace anything with /workers* or /* prepended to just /workers* or /publicPath*
```

Comment:
This is just changing the comment and renaming a variable for clarity no logic changed